editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -907,7 +907,7 @@ void VersionControlEditorPlugin::fetch_available_vcs_plugin_names() {
 }
 
 void VersionControlEditorPlugin::register_editor() {
-	EditorDockManager::get_singleton()->add_dock(version_commit_dock, "", EditorDockManager::DOCK_SLOT_RIGHT_UL);
+	EditorDockManager::get_singleton()->add_dock(version_commit_dock, "", EditorDockManager::DOCK_SLOT_RIGHT_UL, ED_SHORTCUT_AND_COMMAND("docks/open_version_control", TTR("Open Version Control Dock")));
```

Comment:
This shortcut does not appear anywhere, I assume unless you have VCS plugin installed.

Reply:
Yes, the VCS plugin is needed for this to show, just like the `toggle_version_control_bottom_panel` shortcut below.